example/mysite/settings.py

```diff
@@ -23,9 +23,9 @@
 SECRET_KEY = '2^!@63mf-#%l8m_v*o3hz(&2iw35o_6h^3^&x$+1rm558^)_i9'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
+DEBUG = True
 
-ALLOWED_HOSTS = ['127.0.0.1', 'localhost']
+ALLOWED_HOSTS = []
```

Comment:
These changes are me just setting stuff back to the default created by `django-admin startproject`.. these are overridden later.